vpc_ips.go

```diff
@@ -2,9 +2,19 @@ package linodego
 
 import (
 	"context"
+	"fmt"
 )
 
-// ListVPCIPAddresses gets the list of all IP addresses of all VPCs in the Linode account.
-func (c *Client) ListVPCIPAddresses(ctx context.Context, opts *ListOptions) ([]VPCIP, error) {
+// ListAllVPCIPAddresses gets the list of all IP addresses of all VPCs in the Linode account.
+func (c *Client) ListAllVPCIPAddresses(
+	ctx context.Context, opts *ListOptions,
+) ([]VPCIP, error) {
 	return getPaginatedResults[VPCIP](ctx, c, "vpcs/ips", opts)
 }
+
+// ListVPCIPAddresses gets the list of all IP addresses of a specific VPC.
+func (c *Client) ListVPCIPAddresses(
```

Comment:
I wonder if it'd make sense to name these functions something like `ListVPCIPAddresses(...)` and `ListIPAddressesForVPC(...)` (?) to avoid making a breaking changes 🤔

Reply:
I feel "List VPC IP Addresses" and "List IP Addresses for VPC" may have same semantic meaning to some readers, so it might cause confusions.

And since we haven't been having `ListVPCIPAddresses` for a long time, I think it may be fine to make a breaking change? What do you think?

Reply:
Sounds good to me ^